scripts/plot_violine.R

```diff
@@ -219,7 +219,7 @@ ggsave(gg, file = snakemake@output$pdf_count_vs_gene, width = 12, height = 7)
 # sample1_CAGCCCTCAGTA   264  437 sample1_CAGCCCTCAGTA sample1 CAGCCCTCAGTA            100         100 batch1 sample1 0.0389016 0.07551487 0.1144165 0.0228833 0.5102975     1.655303
 
 # saving snakemake meta information into misc slot so all can be exported as one object
-seuratobj@misc <- snakemake
+Misc(object = pbmc_small, slot = "misc")  <- snakemake
```

Comment:
What does this do?

I changed the snakemake object to a list since that's what the seurat object was asking but I haven't doublechecked how it looked like.

Reply:
Seurat version 3 uses “Assay” slots so I think it needs to be specified differently. Their function seems to be recommended for this so they can update it if they change the object in future versions. The test environment on Travis threw an error on this and this fixes it.